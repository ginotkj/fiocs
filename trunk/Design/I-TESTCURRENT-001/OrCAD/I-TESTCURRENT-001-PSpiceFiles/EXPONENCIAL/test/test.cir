** Profile: "EXPONENCIAL-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\exponencial\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-testcurrent-001-pspicefiles/i-testcurrent-001.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 6n 1.5n 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\EXPONENCIAL.net" 


.END
