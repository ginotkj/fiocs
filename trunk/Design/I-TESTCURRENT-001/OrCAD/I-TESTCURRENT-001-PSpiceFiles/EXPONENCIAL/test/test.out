
**** 08/07/10 18:11:22 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "EXPONENCIAL-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\exponencial\


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-testcurrent-001-pspicefiles/i-testcurrent-001.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 6n 1.5n 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\EXPONENCIAL.net" 



**** INCLUDING EXPONENCIAL.net ****
* source I-TESTCURRENT-001
I_EXPONENCIAL         N17870 0 DC 0Adc AC 0Aac
+EXP 0 4m 2n 30p 2.2n 500p
R_R1         N17870 ITEST  1k TC=0,0 
R_R2         0 ITEST  1k TC=0,0 

**** RESUMING test.cir ****
.END

**** 08/07/10 18:11:22 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "EXPONENCIAL-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\exponencial\


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(ITEST)    0.0000  (N17870)    0.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT


    TOTAL POWER DISSIPATION   0.00E+00  WATTS



          JOB CONCLUDED

**** 08/07/10 18:11:22 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "EXPONENCIAL-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\exponencial\


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .02
